samples/json_marshal.go

```diff
@@ -29,6 +29,10 @@ func main() {
 	bytes, _ = json.Marshal(person)
 	fmt.Println(string(bytes))
 
+	person = Person{FirstName: "Madonna", Age: 67}
```

Comment:
this is trivial, but i only just noticed that i just cut and paste this from sting and changed firstname/lastname. I just checked and Madonna is only 60 years old :). Perhaps we just leave this as an easter egg and see if anyone points out it's wrong.

Reply:
Hahaha I like it, I'll give someone a prize if they find it!